lib/paperclip/storage/s3.rb

```diff
@@ -281,6 +282,10 @@ def s3_object(style_name = default_style)
         s3_bucket.object style_name_as_path(style_name)
       end
 
+      def s3_transfer_manager
+        @s3_transfer_manager ||= ::Aws::S3::TransferManager.new
```

Comment:
can we not check if the class/method exists instead of requiring a specific version. This library is being used by many projects, some of which are very old, and we should be careful not to break them.

Reply:
How to proceed with tests? Make also conditional or update them to the latest version of the gem (like in this PR)?